src/main/webapp/courseLoader.js

```diff
@@ -302,10 +292,15 @@
 
   async function passRatingProperties() {
     const urlAndData = await getRatingPropertiesToStore();
+    console.log(urlAndData[1]);
     postRatingProperties(urlAndData[0], urlAndData[1]);
   }
 
   $(function () {
     $('[data-toggle="tooltip"]').tooltip();
   });
+
+  document
+    .getElementById("button")
```

Comment:
update this id when the button id is updated above. In the master I think I called it form-submit so you can choose to match it now or just resolve the conflict later